src/lib/Components/Bidding/Components/Input.tsx

```diff
@@ -51,12 +50,12 @@ export class Input extends Component<InputProps, InputState> {
         border={1}
         borderColor={this.state.borderColor}
         fontSize={3}
-        onBlur={() => this.onBlur()}
-        onFocus={() => this.onFocus()}
-        ref={component => this.props.createCustomRef && this.props.createCustomRef(this.props.refName, component)}
         p={3}
         pb={2}
         {...this.props}
+        ref={this.props.inputRef}
+        onBlur={() => this.onBlur()}
```

Comment:
Ah, just sticking it below the `{...this.props}` is enough to make it override if an `onBlur` is passed in as props? Didn't realize that!